PromisesObjC.podspec

```
@@ -23,6 +23,9 @@ Pod::Spec.new do |s|
   s.public_header_files = "Sources/#{s.module_name}/include/**/*.h"
   s.private_header_files = "Sources/#{s.module_name}/include/FBLPromisePrivate.h"
   s.source_files = "Sources/#{s.module_name}/**/*.{h,m}"
+  s.resource_bundle = {
+    "#{s.name}_Privacy" => 'Sources/Resources/PrivacyInfo.xcprivacy'
```

nit: Sorry, let's actually do module_name here to align with the other podspec.
```suggestion
    "#{s.module_name}_Privacy" => 'Sources/Resources/PrivacyInfo.xcprivacy'
```